xtos/include/sof/trace/preproc-private-inc.h

```diff
@@ -0,0 +1,98 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2016 Intel Corporation. All rights reserved.
+ *
+ * Author: Michal Jerzy Wierzbicki <michalx.wierzbicki@intel.com>
+ */
+
+/* THIS FILE SHOULD NOT BE INCLUDED DIRECTLY */
```

Comment:
hm, so now we have two copies of these rather large headers. These are headers and they aren't included simultaneously of course, they don't affect run-time, etc., but this isn't very good. Why are they needed under posix? Something like testbench or mocka?

Reply:
yes, testbench and cmocka need it from posix path, I have to add it as well. I guess testbench and cmocka build can't be changed to xtos, since the file inside are not same, @singalsu , do you have comments on this question?

Reply:
I hope we could remove need for xtos headers for testbench, topology parser and all other in SOF/tools. The xtos headers like often come with low-level xtensa stuff dependencies and cause difficulty in writing portable code.